{{cookiecutter.repo_name}}/devtools/scripts/initialize_conda.sh

```diff
@@ -0,0 +1,16 @@
+# $CI_OS is $matrix.os, as exported in GHA *.yaml
```

Comment:
This should be in the `gha` directory similar to `travis` I think.

Reply:
Ok! I named it `github-actions` following the scheme in `travis-ci`.